build.sbt

```diff
@@ -137,6 +145,11 @@ lazy val langmeta = crossProject
     langmetaIo,
     langmetaSemanticdb
   )
+  .aggregate(
+    langmetaInputs,
+    langmetaIo,
+    langmetaSemanticdb
+  )
```

Comment:
I aggregated here so I can do `langmeta/publish`, but it's inconsistent with the rest of the build so I'll remove it. I feel like the build could benefit from sprinkling aggregate in some other projects too like `scalameta`. but it's a lot of boilerplate.